hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -783,6 +796,71 @@ public ScmInfo getScmInfo() throws IOException {
     }
   }
 
+  @Override
+  public void transferLeadership(String nodeId)
+      throws IOException {
+    getScm().checkAdminAccess(getRemoteUser());
+    boolean auditSuccess = true;
+    final Map<String, String> auditMap = Maps.newHashMap();
+    auditMap.put("nodeId", nodeId);
+    try {
+      SCMRatisServer scmRatisServer = scm.getScmHAManager().getRatisServer();
+      RaftServer server = scmRatisServer.getDivision().getRaftServer();
```

Comment:
Don't get `RaftServer`.  It only needs a division.

Reply:
Done.